service-automation/src/com/appyschool/services/StaffServices.java

```diff
@@ -14,7 +14,31 @@
 public class StaffServices {
 
 	private static final Logger LOG = LogManager.getLogger(StaffServices.class);
-
+	
+	
+	/**
+	 * Create Designation
+	 * 
+	 * @param payload
+	 * @return
+	 */
+	
+	public static Response createDesignation(String payload) {
```

Comment:
as this is for staff service, adding designation services here doesn't make sense. You need to create this under configuration service.